src/AasxServerBlazor/Pages/Db.razor

```diff
@@ -1,905 +1,803 @@
+                {
+                    var smDBList = db.SMSets.Where(s => s.Identifier.Equals(smidentifier)).ToList();
+                    if (smDBList.Any())
+                    {
+                        var smDB = smDBList.First();
+                        var submodel = ReadDB.GetSubmodel(smDB: smDB);
+
+                        string json =  "";
+                        if (submodel != null)
+                        {
+                            var j = Jsonization.Serialize.ToJsonObject(submodel);
+                            json = j.ToJsonString();
+                        }
+
+                        <span>@json</span>
+                        <br />
+                    }
+                }
+
+                if (url.EndsWith("/sme"))
+                {
+                    List<SMESet> data = PageDB.GetPageSMEData(size, searchLower, smid, smeid);
+                    <span>
+                        &nbsp&nbsp@("SME#: " + data.Count + "/" + db.SMESets.Count() + " size=" + size + " search=" + search + " smid=" + smid + " smeid=" + smeid)
+                    </span>
+                    <br />
+                    <br />
+                    @if (size > 1000 && data.Count > 1000)
+                    {
+                        <b><span style="color:red;">First 1000 as TABLE</span></b>
+                        <br />
+                    }
+                    <table class="table table-bordered table-sm" style="width:100%;word-wrap:break-word;word-break:break-all;">
+                        <thead>
+                            <tr>
+                                <th style="word-break:keep-all">SM#</th>
+                                <th style="word-break:keep-all">Par#</th>
+                                <th style="word-break:keep-all">SME#</th>
+                                <th style="word-break:keep-all">Type</th>
+                                <th style="word-break:keep-all">IdShort</th>
+                                <th style="word-break:keep-all">SemanticId</th>
+                                <th style="word-break:keep-all">Value</th>
+                                <th style="word-break:keep-all">ValueType</th>
+                                <th style="word-break:keep-all">Links</th>
+                                <th style="word-break:keep-all">API</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            @for (int i = 0; i < data.Count && i < 1000; i++)
+                            {
+                                var smeDB = data[i];
+                                string path = smeDB.IdShort;
+                                int? pnum = smeDB.ParentSMEId;
+                                while (pnum != null)
+                                {
+                                    var smeDBP = db.SMESets.Where(s => s.Id == pnum).First();
+                                    path = smeDBP.IdShort + "." + path;

```

Comment:
```suggestion
                                    path = $"{smeDBP.IdShort }.{path}";

```

Please use string concatenation, as this is better to read and it is easier to spot changes.